### PR TITLE
fixed the issue

### LINKED ISSUE #2: Publications
Currently, all of the publications on the main site are in HTML format. We need to move all of the content to JSON files.
- [x] Create a new page called `publications.html`.
- [x] Create a new JSON file called `publications.json`. Load some publications from the main site into it.
- [x] Use the `default` layout and generate data from the `publications.json` file.

Refer to [https://ieee.nitk.ac.in/publications.html](https://ieee.nitk.ac.in/publications.html)

### LINKED ISSUE #6: Team Page
- [x] Faculties can be hard-coded, just like the current implementation.
- [x] Executive member details should be read from the `team.json` file and displayed just like the current implementation.
- [x] If possible, you're free to divide the members into five tabs for five sections - Core, Compsoc Members, Diode Members, Piston Members, Alumni

### PR DESCRIPTION
**Pull request by Shashankhs17**
Please review the changes I have made for the publications issue.

### CODE REVIEW

File: publications.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+layout: default
+title: Publications
+---
+
+
+<div class="colorlib-intro colorlib-bg-white">
+    <div class="container">
+        <div class="row">
+            <div class="col-md-10 col-md-offset-1 colorlib-heading animate-box">
```
Comment (gaurang2001) [inline publications.html]: Since you have `animate-box` for each list item, you don't need it for the entire `div`. Put it for the `h2` tag below though.

File: publications.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+                <h2>IEEE NITK - Papers submitted to coference</h2>
```
Comment (gaurang2001) [inline publications.html]: Have the heading as `List of Papers Submitted to Conferences by IEEE NITK Members`. Same as before.

File: _includes/footer.html [hunk 3]
```diff
@@ -16,12 +16,12 @@ <h4>About IEEE</h4>
                 <h4 class="secondary">Information</h4>
                 <p>
                     <ul class="colorlib-footer-links">
-                        <li><a href="{{ site.url }}"><i class="fa fa-check"></i> Home</a></li>
+                        <li><a href="index.html"><i class="fa fa-check"></i> Home</a></li>
```
Comment (gaurang2001) [inline _includes/footer.html]: have this as "/"

File: _includes/footer.html [hunk 4]
```diff
@@ -16,12 +16,12 @@ <h4>About IEEE</h4>
...
                         <li><a href="{{ site.url }}/team.html"><i class="fa fa-check"></i> Team</a></li>
                         <li><a href="{{ site.baseurl }}/"><i class="fa fa-check"></i> Blog</a></li>
                         <li><a href="{{ site.url }}/gyan/"><i class="fa fa-check"></i> Gyan</a></li>
                         <li><a href="{{ site.url }}/achieve.html"><i class="fa fa-check"></i> Achievements</a></li>
-                        <li><a href="{{ site.url }}/publications.html"><i class="fa fa-check"></i> Publications</a></li>
+                        <li><a href="publications.html"><i class="fa fa-check"></i> Publications</a></li>
```
Comment (gaurang2001) [inline _includes/footer.html:24]: Don't hard-code.

File: _includes/header-default.html [hunk 5]
```diff
@@ -45,6 +45,10 @@
                                     {% endfor %}
                                     </ul>
                                 </li>
+                            {% elsif nav.name == "Publications" %}
+                                <li><a href="publications.html">{{nav.name}}</a></li>
+                            {% elsif nav.name == "Home" %}
+                                <li><a href="index.html">{{nav.name}}</a></li>
```
Comment (gaurang2001) [inline _includes/header-default.html]: Don't hard-code. Revert this change please.

**Review by gaurang2001 — COMMENTED** [review]
After resolving the changes, do put a screenshot of the page in the PR description.

File: _includes/footer.html (hunk 3 above)
Comment (Shashankhs17) [reply _includes/footer.html]: image
changed to " \ "

**Review by gaurang2001 — APPROVED** [review]
Good job🎉

**Comment by Shashankhs17** [comment]
Please review the changes regarding team page issue

image

image

image

image

image

image

**Review by gaurang2001 — APPROVED** [review]
Team page looks good. Good work 🥳
